Compiling error on 22.04/Rolling

```
                                                       
In file included from /opt/ros/rolling/include/rclcpp_lifecycle/rclcpp_lifecycle/lifecycle_node.hpp:996,
                 from /home/steve/Documents/nav2_ws/src/mppic/include/mppic/parameters_handler.hpp:15,
                 from /home/steve/Documents/nav2_ws/src/mppic/src/parameters_handler.cpp:2:
/opt/ros/rolling/include/rclcpp_lifecycle/rclcpp_lifecycle/lifecycle_node_impl.hpp: In member function ‘void mppi::ParametersHandler::getParam(SettingT&, const string&, ParamT, mppi::ParameterType) [with SettingT = bool; ParamT = bool]’:
/opt/ros/rolling/include/rclcpp_lifecycle/rclcpp_lifecycle/lifecycle_node_impl.hpp:256:21: warning: ‘param_in’ may be used uninitialized [-Wmaybe-uninitialized]
  256 |   rclcpp::Parameter param(name, parameter);
      |                     ^~~~~
In file included from /home/steve/Documents/nav2_ws/src/mppic/src/parameters_handler.cpp:2:
/home/steve/Documents/nav2_ws/src/mppic/include/mppic/parameters_handler.hpp:136:10: note: ‘param_in’ was declared here
  136 |   ParamT param_in;
      |          ^~~~~~~~

```

--- a/include/mppic/parameters_handler.hpp
+++ b/include/mppic/parameters_handler.hpp
@@ -133,7 +133,7 @@ template<typename ParamT, typename SettingT, typename NodeT>
 void ParametersHandler::setParam(
   SettingT & setting, const std::string & name, NodeT node) const
 {
-  ParamT param_in;
+  ParamT param_in{};
   node->get_parameter(name, param_in);
   setting = static_cast<SettingT>(param_in);
 }

